azkaban-common/src/test/java/azkaban/project/DirectoryYamlFlowLoaderTest.java

```diff
@@ -37,31 +53,67 @@ public void setUp() {
   }
 
   @Test
-  public void testLoadYamlFileFromDirectory() {
+  public void testLoadBasicYamlFile() {
```

Comment:
test dir without any yaml files?
test dir with yaml file without version specified?
test dir with yaml file with invalid version?

Reply:
Added the test case, thanks!